openssl/src/nid.rs

```diff
@@ -1090,6 +1090,8 @@ impl Nid {
     pub const SHAKE128: Nid = Nid(ffi::NID_shake128);
     #[cfg(ossl111)]
     pub const SHAKE256: Nid = Nid(ffi::NID_shake256);
+    #[cfg(ossl110)]
```

Comment:
And change this to `#[cfg(any(ossl110, libressl271))]`.

Reply:
Done.